### DISCUSSION TITLE
can't run juicefs rmr in  background

### DISCUSSION THREAD

**Discussion by zhoucheng361** [juicedata/juicefs#2248] [General]
**What happened**:
when use & to run juicefs rmr to run as background process, the progress will always run in foreground. 

```
root@bench-tikv-1:~/juicefs2# ./juicefs rmr /tmp/gateway/test3 &
[1] 260859
Removing entries count: 82080   9911.80/s                                 ⠇                                 
Removing entries count: 84634   9932.30/s                                 ⠋                                 
Removing entries count: 233887   8640.86/s   

```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**

**Environment**:
- JuiceFS version (use `juicefs --version`) or Hadoop Java SDK version:
- Cloud provider or hardware configuration running JuiceFS:
- OS (e.g `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Object storage (cloud provider and region, or self maintained):
- Metadata engine info (version, cloud provider managed or self maintained):
- Network connectivity (JuiceFS to metadata engine, JuiceFS to object storage):
- Others:

**Comment by davies** [comment]
When a process is running in background, it can still write to stdout/stderr.

**Comment by SandyXSD** [comment]
Temporary workaround: set `DISPLAY_PROGRESSBAR=false` to disable progress bar.
